icees_api/features/sql.py

```diff
@@ -449,14 +449,34 @@ def normalize_feature(year, feature):
 }
 
 
+def simplify_value(val_str):
+    """
+    simplify value string to integer string if appropriate, e.g., from 1.0 to 1
+    """
+    if isinstance(val_str, str):
```

Comment:
I think this can be simplified so it's a little easier to read:
```
try:
    value_float = float(val_str)
except (ValueError, TypeError):
    return val_str # val_str isn't a number
value_int = int(value_float)
if value_int = value_float:
    return str(value_int)
return val_str
```
Basically, catching the TypeError will replace the first two `if` statements.

Reply:
@maximusunc Thanks for the simplified code snippet. Agreed this is easier to read. Have made the corresponding change.